corona-warn-app/cwa-app-android
# Split check ins midnight (EXPOSUREAPP-5706)

- Split CheckIn across multiple days by midnight UTC time
- Add Unit tests for different scenarios

### Testing 
- Check the [specs](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp/docs/spec/event-registration-client.md#split-checkin-by-midnight-utc) 
- Check test cases
